client/tiny.js

```diff
@@ -19,16 +19,23 @@ var c = {
   subscribers: {}
 };
 
+const status = document.getElementById("status");
+function showStatus(message) {
+  status.textContent = message;
+}
+
 function isError(signal) {
   var isPluginError =
       signal.plugindata &&
       signal.plugindata.data &&
       signal.plugindata.data.success === false;
   return isPluginError || Minijanus.JanusSession.prototype.isError(signal);
-};
+}
 
 function init() {
-  var ws = new WebSocket("ws://localhost:8188", "janus-protocol");
+  const server = `wss://${location.hostname}:8989`;
```

Comment:
Maybe we can make this configurable in the query string if you prefer it to be localhost by default.

Reply:
Query string or textbox on the page is best IMO. I don't want to screw up by forgetting which server a given tab is connecting to, so I want it to be visible.